samples/connect-wpf/WpfConnect.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <OutputType>WinExe</OutputType>
+    <UseWPF>true</UseWPF>
+    <DisableArcade>1</DisableArcade>
```

Comment:
Not sure if there is a better way to handle this. Had some compiler error with Arcade.

Reply:
Curious, what errors with arcade were you seeing?  It _should_ just work by running `build.cmd` once then regular work with `dotnet-interactive.sln`.